hhs_oauth_server/urls.py

```diff
@@ -18,33 +16,19 @@
 ADMIN_REDIRECTOR = getattr(settings, 'ADMIN_PREPEND_URL', '')
 
 urlpatterns = [
-    url(r'^accounts/', include('apps.accounts.urls')),
-    url(r'^connect/userinfo', openidconnect_userinfo,
-        name='openid_connect_userinfo'),
-    url(r'^userinfo', userinfo_w_login,
-        name='openid_connect_user_w_login'),
-    url(r'^register', register, name='dcrp_register'),
     url(r'.well-known/', include('apps.wellknown.urls')),
-    url(r'^protected/bluebutton/fhir/v1/$',
-        fhir_conformance, name='fhir_conformance'),
-    url(r'^protected/bluebutton/fhir/v1/metadata$',
-        fhir_conformance, name='fhir_conformance_metadata'),
-    url(r'^protected/bluebutton/fhir/v1/',
-        include('apps.fhir.bluebutton.urls_oauth')),
-    url(r'^endorsements/', include('apps.dot_ext.endorsementurls')),
-    url(r'^o/', include('apps.dot_ext.urls')),
-
+    url(r'^v1/accounts/', include('apps.accounts.urls')),
+    url(r'^v1/connect/userinfo', openidconnect_userinfo, name='openid_connect_userinfo'),
+    url(r'^v1/userinfo', userinfo_w_login, name='openid_connect_user_w_login'),
```

Comment:
There used to be two versions of some of our API endpoints corresponding to OAuth vs Login. Are these similar? Can this URL be deleted? It doesn't seem to be referenced anywhere.